osler/inventory/migrations/0004_auto_20200827_0831.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.0.5 on 2020-08-27 13:31
+
+from django.db import migrations
+
```

Comment:
Yeah since these migrations are just altering the model, I feel like you could squash the migrations? Like make them all in the same file, you can either manually move it or delete all the migrations and run makemigrations again. Just so you don't have 4 migrations files for your brand new app